src/scenic/core/utils.py

```diff
@@ -163,10 +163,15 @@ def loadMesh(path, filetype, compressed, binary):
 
 
 def unifyMesh(mesh, verbose=False):
-    """Attempt to merge mesh bodies, aborting if something fails.
+    """Attempt to merge mesh bodies, raising an error if something fails.
```

Comment:
Let's mention that it's `ValueError` (are there any other possibilities?) in case someone wants to catch it and do something else.

Reply:
Good point, added!